sdk/search/Azure.Search.Documents/CHANGELOG.md

```diff
@@ -0,0 +1,12 @@
+# Release History
+
+## 1.0.0-preview.2 (Unreleased)
+
+### Breaking Changes
+- Renamed to Azure.Search.Documents (assembly, namespace, and package)
```

Comment:
Nit: for consistency, space after and before headings.

Reply:
The process doc doesn't for the sub-headings: https://azure.github.io/azure-sdk/policies_releases.html#changelog-guidance